@navikt/core/react/src/tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,160 @@
+import React, {
+  forwardRef,
+  HTMLAttributes,
+  useCallback,
+  useRef,
+  useState,
+} from "react";
+import { usePopper } from "react-popper";
+import { Placement } from "@popperjs/core";
+import mergeRefs from "react-merge-refs";
+import cl from "classnames";
+import { Detail, useId } from "..";
+import { useKey, useIsomorphicLayoutEffect } from "react-use";
+
+export interface TooltipProps extends HTMLAttributes<HTMLDivElement> {
+  /**
+   * Element tooltip anchors to
+   */
+  children: React.ReactNode;
+  /**
+   * Open state for contolled tooltip
+   */
+  open?: boolean;
+  /**
+   * Orientation for tooltip
+   * @default "top"
+   */
+  placement?: Placement;
+  /**
+   *  Toggles rendering of arrow
+   *  @default true
+   */
+  arrow?: boolean;
+  /**
+   * Distance from anchor to tooltip
+   * @default 16 w/arrow, 4 w/no-arrow
+   */
+  offset?: number;
+  /**
+   * Tekst shown in tooltip
+   */
+  title: string;
+  /**
+   * Adds a delay when opening to tooltip
+   * @default 150ms
+   */
+  delay?: number;
+}
+
+const Tooltip = forwardRef<HTMLDivElement, TooltipProps>(
+  (
+    {
+      children,
+      className,
+      arrow = true,
+      placement = "top",
+      open,
+      offset,
+      title,
+      delay = 150,
+      id,
+      ...rest
+    },
+    ref
+  ) => {
+    const popoverRef = useRef<HTMLDivElement | null>(null);
+    const mergedRef = mergeRefs([popoverRef, ref]);
+
+    const anchor = useRef<HTMLDivElement | null>(null);
+    const timeoutRef = useRef<number>();
+
+    const [openState, setOpenState] = useState(open ?? false);
+    const tooltipId = useId();
+
+    const handleOpen = () => {
+      if (open !== undefined) return;
+
+      window.clearTimeout(timeoutRef.current);
+      timeoutRef.current = window.setTimeout(() => {
+        setOpenState(true);
+      }, delay);
+    };
+
+    const handleClose = useCallback(() => {
+      window.clearTimeout(timeoutRef.current);
+      if (open !== undefined) return;
+      setOpenState(false);
+    }, [open]);
+
+    useKey("Escape", handleClose, {}, [handleClose]);
+
+    const { styles, attributes, update } = usePopper(
```

Comment:
Er det noen grunn til å ikke bruke `Popover`-komponenten her?

Reply:
En stund siden jeg laget denne nå, men mener det var for å ha full kontroll over open/onClose håndtering.